apps/api-journeys/src/modules/userSession/index.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { testkit, gql } from 'graphql-modules'
+import { schemaBuilder } from '@core/shared/util-graphql'
+import module from '.'
+import { v4 as uuidv4 } from 'uuid'
+import { prismaMock } from '../../lib/mockDb'
+import Journey from '../journey'
+import Block from '../block'
+
+it('creates a journey session', async () => {
+  const app = testkit.testModule(module, {
+    schemaBuilder,
+    modules: [Journey, Block]
+  })
+  const sessionId = uuidv4()
+  const journeyId = uuidv4()
+  prismaMock.userSession.create.mockResolvedValue({ id: sessionId })
+
+  const query = gql`
+    mutation ($journeyId: ID!) {
+      userSessionCreate(journeyId: $journeyId) {
+        id
+      }
```

Comment:
I am surprised that this query doesn't fail. This mutation returns id not { id }. When I try this command against the served graph (`nx serve`) it fails.

```suggestion
      userSessionCreate(journeyId: $journeyId)
```